m01_introduction/02_rst.rst

```diff
@@ -139,11 +139,11 @@ The usual workflow for editing a course is the following:
 1. Open your text editor, a terminal and a web browser.
 2. Edit some RST files in your text editor.
 3. Give command ``./docker-compile.sh`` in the terminal.
-4. Give command ``rm -rf _data && ./docker-up.sh`` in the terminal.
+4. Give command ``./docker-up.sh`` in the terminal.
 5. Go to ``http://localhost:8000/`` in the web browser to view A+
    running on your machine.
 6. Examine the changes you made in A+.
-7. Press ``q`` or ``Ctrl+C`` in the terminal.
+7. Press ``Q`` or ``Ctrl+C`` in the terminal.
```

Comment:
New functionality with ESC and `S` could be documented here. Not required for this PR, but you can add if you like.